docs/migration-guide.md

```diff
@@ -0,0 +1,46 @@
+# Migration guide
+
+Libvalkey can replace both libraries `hiredis` and `hiredis-cluster`.
+This guide highlights which APIs that have changed and what you need to do when migrating to libvalkey.
+
+The general actions needed are:
+
+* Replace the prefix `redis` with `valkey` in API usages.
+* Update include paths depending on your previous installation.
+  All `libvalkey` headers are now found under `include/valkey/`.
+
+## Migrating from `hiredis` v1.2.0
+
+The type `sds` is removed from the public API.
+
+### Removed API functions
+
+* `valkeyFormatSdsCommandArgv` removed from API. Can be replaced with `valkeyFormatCommandArgv`.
+* `valkeyFreeSdsCommand` removed since the `sds` type is for internal use only.
```

Comment:
Hiredis 1.2.0 doesn't have a function called `valkeyFormatSdsCommandArgv`.

```suggestion
* `redisFormatSdsCommandArgv` removed from API. Can be replaced with `valkeyFormatCommandArgv`.
* `redisFreeSdsCommand` removed since the `sds` type is for internal use only.
```

Reply:
of course! Good finding